content/en/blog/_posts/2017-02-00-Run-Deep-Learning-With-Paddlepaddle-On-Kubernetes.md

```diff
@@ -3,6 +3,9 @@ title: " Run Deep Learning with PaddlePaddle on Kubernetes "
 date: 2017-02-08
 slug: run-deep-learning-with-paddlepaddle-on-kubernetes
 url: /blog/2017/02/Run-Deep-Learning-With-Paddlepaddle-On-Kubernetes
+author: >
+  Yi Wang ([Baidu Research](http://research.baidu.com/)),
+  Xiang Li ([CoreOS](https://coreos.com/))
 ---
 
 _Editor's note: Today's post is a joint post from the deep learning team at Baidu and the etcd team at CoreOS._  
```

Comment:
I'd drop this note. We now just credit the person and optionally their organization.